src/Controller/TestController.php

```diff
@@ -1,23 +1,21 @@
 <?php
```

Comment:
We don't need this anymore, do we?  It was just something I was playing with in development.  It could be removed from the  code and the routing config.